lib/matplotlib/cbook.py

```diff
@@ -708,6 +708,9 @@ def is_string_like(obj):
         return False
     return True
 
+def is_list_like(obj):
```

Comment:
I'm not sold on the need for this - the test is pretty lightweight. Having said that, I don't really agree with the is_string_like method either, though that does do a bit more work.

If this method is going to stay, we need to appease the PEP8 gods and put 2 newlines before and after its definition. :smile: